src/incidents/util/extractUsername.ts

```diff
@@ -0,0 +1,2 @@
+export const extractUsername = (username: string) =>
```

Comment:
this util can probably used in several different spots. To indicate that, I think this is better positioned in `src/shared/utils` folder